Nest.js 10，安装依赖失败

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: rational-use@0.0.1
npm ERR! Found: @nestjs/common@10.0.3
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^10.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^8.4.0 || ^9.0.0" from nestjs-soap@3.0.1
npm ERR! node_modules/nestjs-soap
npm ERR!   nestjs-soap@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nestjs-soap",
-  "version": "3.0.1",
+  "version": "3.0.2",
   "description": "Nestjs module wrapper for soap",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
@@ -41,7 +41,7 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "peerDependencies": {
-    "@nestjs/common": "^8.4.0 || ^9.0.0"
+    "@nestjs/common": "^8.4.0 || ^9.0.0 || ^10.0.0"
   },
   "dependencies": {
     "soap": "^1.0.0"

